README.md

```diff
@@ -36,6 +36,7 @@ Supported languages
 * **Fish Shell** ([*fish_indent*](https://fishshell.com/docs/current/commands.html#fish_indent))
 * **Fortran 90** ([*fprettify*](https://github.com/pseewald/fprettify))
 * **Go** ([*gofmt*](https://golang.org/cmd/gofmt/))
+* **Gleam** ([*gleam format*](https://gleam.run/))
```

Comment:
Please swap `Go` and `Gleam` to keep the list alphabetically sorted.

Reply:
derp. Letters are difficult. :facepalm:

Reply:
No problem. Thanks!